test.js

```diff
@@ -60,7 +59,7 @@ define(['hermes', 'chai'], function (hermes, chai) {
             });
 
             describe('a notification', function () {
-                it('should create a list item', function(){
+                it('should create a list item', function () {
```

Comment:
This was prior to our newer practice of not using "should" in tests.

Or is it whitespace? My repo, my rules. ;)